tfhe/c_api_tests/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ endif()
 set(TFHE_C_API_RELEASE "${CMAKE_CURRENT_SOURCE_DIR}/../../target/${CARGO_PROFILE}")
 
 option(WITH_FEATURE_GPU "Enable if tfhe-rs C API was compiled with the 'gpu' feature activated" OFF)
+option(WITH_FEATURE_GPU "Enable if tfhe-rs C API was compiled with the 'forward_compatibility' feature activated" OFF)
```

Comment:
Variable name should be `WITH_FORWARD_COMPATIBILITY`

Reply:
thanks indeed 😱